public/utils/contextMenu/actions.tsx

```diff
@@ -81,7 +81,7 @@ export const getActions = () =>
       title: i18n.translate('dashboard.actions.alertingMenuItem.addAlertingMonitor.displayName', {
         defaultMessage: 'Add alerting monitor',
       }),
-      icon: 'plusInCircle' as EuiIconType,
+      icon: 'plus' as EuiIconType,
```

Comment:
Do you have an screenshot of this one? Think its fine, but don't think it was in the summary

Reply:
Edited with correct screenshots